src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1333,9 +1333,37 @@ def post_process(self, r):
                 while r is not None and r["properties"]["provisioningState"].lower() == "inprogress":
                     r = self.client.show(self.cmd, self.get_argument_resource_group_name(), self.get_argument_name())
                     time.sleep(1)
-                linker_client.linker.begin_create_or_update(resource_uri=r["id"],
-                                                            parameters=item["parameters"],
-                                                            linker_name=item["linker_name"]).result()
+
+                service_connectors_def_list, service_bindings_def_list = parse_service_bindings(
+                    self.cmd, self.get_argument_service_bindings(), self.get_argument_resource_group_name(), self.get_argument_name())
+                self.set_argument_service_connectors_def_list(
+                    service_connectors_def_list)
+
+                # Case: Kafka on confluent cloud with two linker operations.
+                if any(len(item["parameters"]) ==
+                       2 for item in service_connectors_def_list):
+                    service_connectors_def_list = update_connectors_with_two_parameters(
+                        item, service_connectors_def_list.copy(), r["id"])
+                # Validate uniqueness across all bindings
+                check_bindings_and_raise_error(
```

Comment:
seems `check_bindings_and_raise_error` has been done in `set_up_service_binds`, can I ask is it required to do it again?

Reply:
two sets of bindings and the collective needs to be a set.